corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -596,6 +601,7 @@ class RemoteRequestSession(OrderedXmlObject, XmlObject):
 
     queries = NodeListField('query', RemoteRequestQuery)
     data = NodeListField('datum', SessionDatum)
+    instance_data = NodeListField('instance-datum', InstanceDatum)
```

Comment:
I'm not sure if this is necessary/desirable. Nothing references it. But is _is_ part of the session, and I generally think the xml_models classes should be complete definitions.